pkg/frontend/adminactions/kubeactions.go

```diff
@@ -69,9 +78,15 @@ func NewKubeActions(log *logrus.Entry, env env.Interface, oc *api.OpenShiftClust
 
 		dyn:       dyn,
 		configcli: configcli,
+		kubecli:   kubecli,
 	}, nil
 }
 
+func (k *kubeActions) KubeGetPodLogs(ctx context.Context, namespace, name, containerName string) ([]byte, error) {
```

Comment:
The `name` variable should be changed to `pod` or `podName` here as well.

Reply:
done ..